change_quotes.py

```diff
@@ -444,7 +464,7 @@ def replace_quotes(self, region, replacement):
         debug("Replace: %s with %s" % (self.view.substr(region), replacement))
         self.view.replace(self.edit, region, replacement)
 
-    def escape_unescape(self, region, quote, replacement):
+    def escape_unescape(self, region, quote, replacement, replacement_re = None):
```

Comment:
Also, I see both `replacement` and `quote` can now be None, would love to also see that mentioned in the docstring :)